db/migrate/20160131182301_add_download_policy_to_katello_repositories.rb

```diff
@@ -0,0 +1,6 @@
+class AddDownloadPolicyToKatelloRepositories < ActiveRecord::Migration
+  def change
+    add_column :katello_repositories, :download_policy, :string
+    update "UPDATE katello_repositories SET download_policy = 'on_demand' WHERE content_type = 'yum'"
```

Comment:
Could this be done through the ORM? We recently had a similar discussion around this on https://github.com/Katello/katello/pull/5709#discussion_r49897802